controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/StreamRequestHandler.java

```diff
@@ -65,31 +65,51 @@ public boolean toPostpone(ControllerEvent event, long pickupTime, Throwable exce
 
     @Override
     public CompletableFuture<Void> processScaleOpRequest(ScaleOpEvent scaleOpEvent) {
+        log.info("Processing scale request {} for stream {}/{}", scaleOpEvent.getRequestId(), scaleOpEvent.getScope(), scaleOpEvent.getStream());
```

Comment:
Here's a tip to make your code more compact:
- Make ScaleOpEvent, UpdateStreamEvent, TruncateStreamEvent, etc. implement a new, common interface, which exposes RequestId, Scope and Stream.
- Make 2 private methods (logBeginProcessing/logEndProcessing) which take an arg of this interface and log it from there. It will save you from having to include so many similar variants of the same log calls all over this class.

Reply:
i agree, but i am trying to keep changes to a minimum. just adding some useful logging in a single class for now.